content/using_bookwyrm/groups.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: Groups
+Date: 2025-05-26
+Order: 12
+---
+
+You can join or create a Group with others on Bookwyrm.
+There are many things a group could be made for - a book club, a bunch of friends, whatever makes sense for you.
+Groups can curate book **Lists** together, but that's about it right now.
+Currently, the Bookwyrm developers are focusing on other things, not groups, but they will be improved in the future.
+
+To find the groups page, first navigate to your profile, then open the 'Groups' tab.
+
+## Group administration
+
+### Creating a group
+
+On the top right of the groups page is the 'Create group' button. 
+You will then need to give it a name and describe what it is about, before choosing its privacy setting. 
+
+### Group privacy
+
+Groups have the same privacy settings as posts and lists do, except they can't be Followers-only.
+Group membership always requires an invitation from the group's owner, even if it is marked Public.
+For more details, see [here](/privacy-controls#Groups).
+
+### Adding and removing users
+
+Groups are currently limited only to members on the current instance.
+To add members, search their username in the search bar, then click 'Invite'.
+To remove members, click the 'Remove' button and then 'Confirm'.
+
+## Group membership
+
+### Joining a group
+
+To join a group, you must be on the same instance (website) and be invited by its owner. 
+You will then receive a notification which lets you join.
+The group owner is marked with a star symbol.
+
+### Using group Lists
+
+Group Lists work very similarly to normal Lists, except they are curated by all members of a Group.
+They can be created by any member of a group from the group's homepage.
+
+## Future improvements
```

Comment:
I don't disagree with any of this but it seems a bit strange to add _suggestions_ for improvement in the docs. The "related ideas" is useful to leave here though.

Reply:
easy to accidentally do when you're on a roll!